### PR TITLE
Setup terraform

### PR DESCRIPTION
**Pull request by YakMan101**
Add the terraform folder to main branch. 
The terraform files work for the short term pipeline, long term and dashboard has yet to be implemented. 

Note: Environment variables will need to be specified for terraform after ETL pipeline scripts are containerised and uploaded to ECR repos.

### CODE REVIEW

File: terraform/.gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Environment variables
+.terraform
+.terraform.lock.hcl
+terraform.*
+.terraform.tfstate.lock.info
+.pemkey
+.env
```
Comment (Leon-JS02) [inline terraform/.gitignore]: Maybe *.env, and *.pem should be added for more security?

**Review by mehrdadhalali — APPROVED** [review]
LGTM

File: terraform/readme.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Variables to fix
```
Comment (Leon-JS02) [inline terraform/readme.md]: Very minor, but I think it's conventionally uppercase `README.me`

File: terraform/sample_image/test.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import json
+from os import environ as ENV
+from datetime import datetime
```
Comment (Leon-JS02) [inline terraform/sample_image/test.py:4]: Very minor: imports should be divided up by third party, standard library, and local scripts.

**Review by Leon-JS02 — CHANGES_REQUESTED** [review]
Very solid TF work, just a few minor changes for cleanliness.

**Review by Leon-JS02 — APPROVED** [review]
Nice, LGTM.

**Pull request merged**